pyproject.toml

```diff
@@ -11,7 +11,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = ">=3.7"
-numpy = ">=1.10.1,<2.0.0"
+numpy = ">=2.0.0"
```

Comment:
This could even remain `numpy = ">=1.26.0"`, since all changes should work with both NumPy 1.x and NumPy 2.x.